src/components/Typography/Typography.vue

```diff
@@ -30,20 +30,7 @@ export default defineComponent({
 		 * Html tag
 		 */
 		tag: {
-			type: String as PropType<
-				| 'p'
-				| 'span'
-				| 'h1'
-				| 'h2'
-				| 'h3'
-				| 'h4'
-				| 'h5'
-				| 'h6'
-				| 'legend'
-				| 'label'
-				| 'li'
-				| 'div'
-			>,
+			type: String as PropType<typeof typographyHtmlTags[number]>,
```

Comment:
Não pode fazer isso por questão de como o storybook gera a documentação.
Rodei a branch e ficou assim:
![image](https://github.com/Farm-Investimentos/front-mfe-components/assets/84783765/e10010a2-da3d-4a17-b511-a390ac4ecf60)

o storybook renderiza direto o que está inline nas props, ele não é inteligente em ver se é uma variável e renderizar o conteúdo. Por isso, se vc for ver, todos componentes tem as PropType definidas assim.

Reply:
@waverapha na documentação online [vc consegue ver](https://storybook.plataforma.portalfarm.com.br/index.html?path=/docs/typography-atom--primary)

![image](https://github.com/Farm-Investimentos/front-mfe-components/assets/84783765/20498bbe-f7d3-4c23-b267-8b1fd79b9658)

Reply:
Putz, que pena que não tem como, vou só adicionar as tags que quero então